internal/cli/atlas/iam/organizations/invitations/invitations.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package invitations
+
+import (
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
+	const use = "invitations"
+	cmd := &cobra.Command{
+		Use:     use,
+		Short:   "Invitation operations.",
+		Long:    "Create, list and manage your MongoDB project invites.",
+		Aliases: cli.GenerateAliases(use),
+	}
+
+	cmd.AddCommand(
+		ListBuilder(),
+		DescribeBuilder(),
+		DeleteBuilder(),
+		UpdateBuilder(),
+		InviteBuilder())
+	return cmd
+}
```

Comment:
Could you check that these commands are not added to MongoCLI as well?

Reply:
Yep, only added to the AtlasCLIBuilder

https://github.com/mongodb/mongodb-atlas-cli/blob/843495b5959a45a42ee6911dee747b32eb9ce30f/internal/cli/iam/organizations/organizations.go#L56-L63